AFOAuth2Manager/AFOAuth2Manager.h

```diff
@@ -18,22 +18,17 @@
 // AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
+// THE SOFTWARE
 
 #import <Foundation/Foundation.h>
+#import "AFOAuthCredential.h"
+#import "AFHTTPRequestSerializer+OAuth2.h"
 
-#import <AFNetworking/AFHTTPRequestOperationManager.h>
+@import AFNetworking;
```

Comment:
Can this be switched back to `#import <AFNetworking/AFHTTPSessionManager.h>` in order to maintain compatibility with CocoaPods setups that don't use dynamic frameworks? 

 The change was made in commit 42744f9ce61e12dc641867072bc8d79e145659ec when adding a test target. It's not clear to me why the change was required. The tests complete with the `#import <>` style import.

Somewhat related to issue https://github.com/AFNetworking/AFOAuth2Manager/pull/114